packages/instantsearch.js/src/widgets/hits-per-page/__tests__/hits-per-page.test.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * @jest-environment jsdom
+ */
+/** @jsx h */
+import { createSearchClient } from '@instantsearch/mocks';
+
+import instantsearch from '../../../index.es';
+import hitsPerPage from '../hits-per-page';
+
+describe('hitsPerPage', () => {
+  describe('options', () => {
+    test('throws without a `container`', () => {
+      expect(() => {
+        const searchClient = createSearchClient();
+
+        const search = instantsearch({
+          indexName: 'indexName',
+          searchClient,
+        });
+
+        search.addWidgets([
+          hitsPerPage({
+            // @ts-expect-error
+            container: undefined,
+          }),
+        ]);
+      }).toThrowErrorMatchingInlineSnapshot(`
+"The \`container\` option is required.
+
+See documentation: https://www.algolia.com/doc/api-reference/widgets/hits-per-page/js/"
+`);
+    });
+
+    test('warns with an item without a `value`', () => {
```

Comment:
is this warning only in the InstantSearch.js widget? maybe should be moved to connector

Reply:
Yes, makes sense, I did it in 9d0ab94ab.

Later on we should migrate all connector tests to connector common test suites. Won't do it here because it would exceed the allocated time for the task, but that should be the next step once we're done with widgets.